### ISSUE TITLE
Missing fields 

### ISSUE THREAD

**Issue by XX-Yin** [AllenNeuralDynamics/aind-data-schema#918]
**Describe the bug**
1. Missing `Laser` in `STIMULUS_DEVICES` (rig.py)
2. Missing `Fujinon` in `LENS_MANUFACTURERS` (organizations.py)
3. Missing organization `Midwest Optical Systems. Inc.` in `Organization` (organization.py)

**To Reproduce**
5. Scroll down to '....'
6. See error

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by saskiad** [comment]
The Laser should be specified in the `light_source_config` field in `StimulusEpoch` and not be listed as a `stimulus_device`

**Comment by saskiad** [comment]
item 3 is a duplicate of #915

**Comment by XX-Yin** [comment]
> `light_source_config`

I hope to add the `optogenetics lasers` to the `stimulus_devices`. And the `Laser` is not included in the `STIMULUS_DEVICES`.  
![image](https://github.com/AllenNeuralDynamics/aind-data-schema/assets/109394934/15a40c33-f2e6-4629-88f7-ab668adfb691)

![image](https://github.com/AllenNeuralDynamics/aind-data-schema/assets/109394934/f23ab56b-53c2-4bd5-b411-34fe8151321e)

**Comment by XX-Yin** [comment]
adding other missing fields 

Missing `wavelength` and `wavelength_unit` in `LightEmittingDiodeConfig` (session.py)

**Comment by saskiad** [comment]
Ah, sorry, I thought you were talking about the Session. The laser should be listed in the `light_sources` field for the rig.

**Comment by saskiad** [comment]
Does the LED not have a fixed wavelength?

**Comment by XX-Yin** [comment]
> Ah, sorry, I thought you were talking about the Session. The laser should be listed in the `light_sources` field for the rig.

Yes, the laser is listed in the `light_sources` field. Just to confirm that `lasers` used for optogenetics should go to `stimulus_devices` instead of the `light_sources` ?  And the `light_sources` is usually used for lights used for imaging purpose. Do I understand it correctly?

**Comment by XX-Yin** [comment]
> Does the LED not have a fixed wavelength?

Typically, it is a normal distribution (shown below). 

![image](https://github.com/AllenNeuralDynamics/aind-data-schema/assets/109394934/4d3afcee-de05-4db7-b1ee-f0c72573357e)

There are 3 parameters that are most useful. `Wavelength` , `Peak wavelength range` and the `Spectrum half width`. I think the last two could be optional.  
![image](https://github.com/AllenNeuralDynamics/aind-data-schema/assets/109394934/4cc11438-e925-4a4c-b664-9f482dcbd4b3)

**Comment by saskiad** [comment]
No. This is understandably confusing and I'll try to add better documentation: 

For the Rig, all lasers go in `light_sources`. 
In the Session, lasers used for optogenetics are listed in the `light_source_config` of the `StimulusEpoch`. Lasers used in calcium imaging are listed in `light_sources` of the `DataStream`

**Comment by XX-Yin** [comment]
> The wavelength information is a fixed property of the LED. So that is documented in the Rig.

Okay, got it. I will delete the fourth point.

**Comment by XX-Yin** [comment]
> The wavelength information is a fixed property of the LED. So that is documented in the Rig.

One related question is the `wavelength` is required for `LaserConfig` but not for `LightEmittingDiodeConfig` in the `session.py`

**Comment by saskiad** [comment]
> > The wavelength information is a fixed property of the LED. So that is documented in the Rig.
> 
> One related question is the `wavelength` is required for `LaserConfig` but not for `LightEmittingDiodeConfig` in the `session.py`

There are lasers where you can adjust the wavelength, so we need to be able to track what was the wavelength that was used.

**Comment by XX-Yin** [comment]
> No. This is understandably confusing and I'll try to add better documentation:
> 
> For the Rig, all lasers go in `light_sources`. In the Session, lasers used for optogenetics are listed in the `light_source_config` of the `StimulusEpoch`. Lasers used in calcium imaging are listed in `light_sources` of the `DataStream`

Thanks for making it clear. I will delete the first point.
